indico/modules/events/layout/default.py

```diff
@@ -97,9 +97,9 @@ def get_default_menu_entries():
             visible=_visibility_call_for_abstracts,
         ),
         MenuEntryData(
-            title=_("View my Abstracts"),
-            name='user_abstracts',
-            endpoint='event.userAbstracts',
+            title=_("My Abstracts"),
+            name='my_abstracts',
```

Comment:
I wouldn't rename this for now. It just adds another step (updating all menu items with the old name in the DB) to the migration.